code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -242,7 +242,7 @@
 /datum/gear/accessory/detbadge
 	display_name = "badge, detective"
```

Comment:
Should be badge, investigator then. Does not matter for the coat imho, but should be changed for their badge to reflect it ICly

Reply:
This is just the name in the loadout, and I do not think it is important. The actual badge simply says that it is 'marks the holder as an investigative agent'.

Reply:
I think changing the loadout names would improve user-friendliness for new players, immediately alerting them to the fact that we do not have a separate Detective role.